client/post-editor/editor-diff-viewer/style.scss

```diff
@@ -22,10 +22,12 @@
 }
 
 .editor-diff-viewer__additions {
-	background: #e9f9fe;
+	background: #e8f9fe;
```

Comment:
@frontdevde, what do you think about making these values relative to the variables we're using as the border?

e.g. `lighten( $blue-wordpress, 57% )` is fairly close to the `#e8f9fe` value here.

Reply:
Happy to do it that way if it's preferred! :)

Personally, I'm not a fan of using `lighten` directly. It opens the door for a lot of different color variations throughout the code base. Ideally, I like the idea of defining even the `lighten`/`darken` shades of the colors in the colors.scss. This way we would see in one place which variations are already used and where we can consolidate. 

But that's a bigger discussion for another day. Having some kind of variable use here is definitely better than using a fixed color value. 👍

Reply:
>  like the idea of defining even the lighten/darken shades of the colors in the colors.scss.

Agreed. Would be happy to see these codified as such in a follow-up PR.